packages/0x.js/package.json

```diff
@@ -12,11 +12,11 @@
   "main": "lib/src/index.js",
   "types": "lib/src/index.d.ts",
   "scripts": {
-    "prebuild": "npm run clean",
+    "prebuild": "run-s clean generate_contract_wrappers",
     "build": "run-p build:umd:prod build:commonjs; exit 0;",
     "docs:json": "typedoc --excludePrivate --excludeExternals --target ES5 --json $JSON_FILE_PATH $PROJECT_DIR",
     "upload_docs_json": "aws s3 cp generated_docs/index.json $S3_URL --profile 0xproject --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers --content-type application/json",
-    "generate_contract_wrappers": "abi-gen --abiGlob 'src/artifacts/@(Exchange|Token|TokenTransferProxy|EtherToken|TokenRegistry).json' --templates ../abi-gen-templates/ --output src/contract_wrappers/generated --fileExtension ts",
+    "generate_contract_wrappers": "node ../abi-gen/lib/index.js --abiGlob 'src/artifacts/@(Exchange|Token|TokenTransferProxy|EtherToken|TokenRegistry|DummyToken).json' --templates contract_templates --output src/contract_wrappers/generated",
```

Comment:
Why isn't `abi-gen` a dev-dependency of this subpackage? I think it's a slippery slope to start using other packages by way of the directory system. In theory we should be able to separate the packages into independent repos at any time and still have them work.

Reply:
It is a dev-dependency. Check the file. It's the CI yarn problem that I've told you about before. I didn't find a way around it. On my local machine the same works without a hack, but on CI yarn hoists the dependency and doesn't create a binary link.

Reply:
Ok, then let's leave a comment here. It is a bit janky but seems like the suggested approach (https://stackoverflow.com/questions/14221579/how-do-i-add-comments-to-package-json-for-npm-install):

```
"//": "HACK: We typically do not link directly between subpackages but due to a CircleCI Yarn issue, made any exception here.",
"generate_contract_wrappers": "node ../abi-gen/lib/index.js --abiGlob 'src/artifacts/@(Exchange|Token|TokenTransferProxy|EtherToken|TokenRegistry|DummyToken).json' --templates contract_templates --output src/contract_wrappers/generated",
```